documentation/modules/configuring/ref-storage-jbod.adoc

```diff
@@ -68,10 +68,19 @@ Where `_id_` is the ID of the volume used for storing data for Kafka broker pod
 
 == Configuring the storage volume used to store the KRaft metadata log
 
-When running a Kafka cluster in a KRaft mode, each node must store the KRaft metadata log on one of its data volumes.
-By default, the KRaft metadata log is stored on the volume with the lowest ID.
+In KRaft mode, a Kafka cluster stores its metadata in a log present on all nodes, including brokers and controllers.
```

Comment:
I wonder if this would be a bit better wording as not every metadata log is equal and on the brokers it is AFAIU only a local copy and not the main metadata log as with the controllers.

```suggestion
In KRaft mode, a copy of the metadata log is stored on every node, including brokers and controllers.
```